miner/block.go

```diff
@@ -128,7 +128,19 @@ func prepareBlock(w *worker) (*blockState, error) {
 		if (lastCommitment != common.Hash{}) {
 			lastRandomnessParentHash := rawdb.ReadRandomCommitmentCache(w.db, lastCommitment)
 			if (lastRandomnessParentHash == common.Hash{}) {
-				return b, errors.New("Failed to get last randomness cache entry")
+				log.Warn("Randomness cache miss while building a block. Attempting to recover.")
```

Comment:
```suggestion
				log.Warn(fmt.Sprintf("Randomness cache miss while building block %d. Attempting to recover.", header.Number.Uint64()))
```

Reply:
I've added the block number to this log line in 262bf2cc3508cde195fd8585ba7c6542b2b6d72d. I used the variable length args method that is supported by this logging package.